frontend/src/pages/login.js

```diff
@@ -37,8 +37,18 @@ const Login = () => {
         <Grid.Column>
           <Tab
             panes={[
-              { menuItem: 'Sign up', render: () => <SignUpForm /> },
-              { menuItem: 'Log in', render: () => <SignInForm /> },
+              {
+                menuItem: 'Sign up',
+                render: function SignUpTab() {
+                  return <SignUpForm />
+                },
+              },
+              {
+                menuItem: 'Log in',
+                render: function SignInTab() {
+                  return <SignInForm />
+                },
+              },
```

Comment:
Why is that being changed? Seems clearer and more compact before.

Reply:
There's a rule that all components should have a display name. This is the error when using anonymous functions
```
error  Component definition is missing display name  react/display-name
```

Reply:
Wait a minute, can't this just be: `render: SignUpform,` and `render: SignInForm,`?

Reply:
I tried this the first time working on this page and it doesn't work.
![image](https://user-images.githubusercontent.com/37152329/122398003-c0f1da00-cf79-11eb-85c3-0193fcdc5652.png)
and the attributes of the `pane` prop
![image](https://user-images.githubusercontent.com/37152329/122398231-fa2a4a00-cf79-11eb-8c05-16731b7b4d6e.png)

Reply:
Those docs don't really explain why it wouldn't work. What actually happened when you tried it? It's not that important anyway, just leave it.

Reply:
> Those docs don't really explain why it wouldn't work

The `render` value should be a function that returns a component, not a component.

> What actually happened when you tried it?
Error with calling useEffect outside of React component.